src/cljs/witan/ui/components/icons.cljs

```diff
@@ -139,6 +142,50 @@
   [& args]
   (apply create-key "play_arrow" args))
 
+(defn bug
+  [& args]
+  (apply create-key "bug_report" args))
+
+(defn request-to-share
+  [& args]
+  (apply create-key "assignment_return" args))
+
+(defn user
+  [& args]
+  (apply create-key "person" args))
+
+(defn organisation
+  [& args]
+  (apply create-key "location_city" args))
+
+(defn delete
+  [& args]
+  (apply create-key "delete" args))
+
+(defn tick
+  [& args]
+  (apply create-key "done" args))
+
+(defn schema
+  [& args]
+  (apply create-key "assignment" args))
+
+(defn email
+  [& args]
+  (apply create-key "email" args))
+
+(defn tick-circle
+  [& args]
+  (apply create-key "check_circle" args))
+
+(defn warning
```

Comment:
I have consciously abstracted away the MDI names - in some cases they're close but in some cases they're very far apart. For example, `icons/request-to-share` uses "assignment_return". It's not at all obvious why you'd need/want to use "assignment_return" everywhere you want the icon that represents 'request-to-share'.